[2024-01-30T02:52:50.820+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [queued]>
[2024-01-30T02:52:50.833+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [queued]>
[2024-01-30T02:52:50.840+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 6
[2024-01-30T02:52:50.877+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): local_job_raw_to_formatted> on 2024-01-08 00:00:00+00:00
[2024-01-30T02:52:50.891+0000] {standard_task_runner.py:60} INFO - Started process 18216 to run task
[2024-01-30T02:52:50.909+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'local_job_raw_to_formatted', 'scheduled__2024-01-08T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmp11u5_emm']
[2024-01-30T02:52:50.912+0000] {standard_task_runner.py:88} INFO - Job 177: Subtask local_job_raw_to_formatted
[2024-01-30T02:52:51.036+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T02:52:51.227+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='local_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-08T00:00:00+00:00'
[2024-01-30T02:52:51.247+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-30T02:52:51.249+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake
[2024-01-30T02:52:56.131+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SparkContext: Running Spark version 3.5.0
[2024-01-30T02:52:56.136+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SparkContext: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T02:52:56.137+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SparkContext: Java version 17.0.9
[2024-01-30T02:52:56.266+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-01-30T02:52:56.438+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceUtils: ==============================================================
[2024-01-30T02:52:56.440+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-01-30T02:52:56.443+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceUtils: ==============================================================
[2024-01-30T02:52:56.443+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SparkContext: Submitted application: RawToFormatted
[2024-01-30T02:52:56.497+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-01-30T02:52:56.518+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceProfile: Limiting resource is cpu
[2024-01-30T02:52:56.519+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-01-30T02:52:56.661+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SecurityManager: Changing view acls to: juniortemgoua0
[2024-01-30T02:52:56.662+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SecurityManager: Changing modify acls to: juniortemgoua0
[2024-01-30T02:52:56.663+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SecurityManager: Changing view acls groups to:
[2024-01-30T02:52:56.665+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SecurityManager: Changing modify acls groups to:
[2024-01-30T02:52:56.666+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:56 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: juniortemgoua0; groups with view permissions: EMPTY; users with modify permissions: juniortemgoua0; groups with modify permissions: EMPTY
[2024-01-30T02:52:57.400+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO Utils: Successfully started service 'sparkDriver' on port 36263.
[2024-01-30T02:52:57.580+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO SparkEnv: Registering MapOutputTracker
[2024-01-30T02:52:57.738+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO SparkEnv: Registering BlockManagerMaster
[2024-01-30T02:52:57.800+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-01-30T02:52:57.804+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-01-30T02:52:57.810+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-01-30T02:52:57.839+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-800fc713-02b9-40c5-bda3-73459969ae23
[2024-01-30T02:52:57.861+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-01-30T02:52:57.899+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:57 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-01-30T02:52:58.157+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-01-30T02:52:58.416+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-01-30T02:52:58.608+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO SparkContext: Added JAR file:/home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar at spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:36263/jars/spark_job_2.12-0.1.0.jar with timestamp 1706583176112
[2024-01-30T02:52:58.903+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Executor: Starting executor ID driver on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T02:52:58.906+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Executor: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T02:52:58.908+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Executor: Java version 17.0.9
[2024-01-30T02:52:58.933+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-01-30T02:52:58.936+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:58 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@6ed043d3 for default.
[2024-01-30T02:52:59.068+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO Executor: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:36263/jars/spark_job_2.12-0.1.0.jar with timestamp 1706583176112
[2024-01-30T02:52:59.252+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO TransportClientFactory: Successfully created connection to instance-1.europe-west9-a.c.data-lake-project-409321.internal/10.200.0.5:36263 after 135 ms (0 ms spent in bootstraps)
[2024-01-30T02:52:59.271+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO Utils: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:36263/jars/spark_job_2.12-0.1.0.jar to /tmp/spark-a69acf62-f2be-4860-84e6-dc5c998198d7/userFiles-b773106a-1c52-461e-91b0-5bd135ee1866/fetchFileTemp458937980513526927.tmp
[2024-01-30T02:52:59.396+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO Executor: Adding file:/tmp/spark-a69acf62-f2be-4860-84e6-dc5c998198d7/userFiles-b773106a-1c52-461e-91b0-5bd135ee1866/spark_job_2.12-0.1.0.jar to class loader default
[2024-01-30T02:52:59.428+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 33597.
[2024-01-30T02:52:59.429+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO NettyBlockTransferService: Server created on instance-1.europe-west9-a.c.data-lake-project-409321.internal:33597
[2024-01-30T02:52:59.431+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-01-30T02:52:59.447+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 33597, None)
[2024-01-30T02:52:59.450+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO BlockManagerMasterEndpoint: Registering block manager instance-1.europe-west9-a.c.data-lake-project-409321.internal:33597 with 434.4 MiB RAM, BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 33597, None)
[2024-01-30T02:52:59.456+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 33597, None)
[2024-01-30T02:52:59.460+0000] {spark_submit.py:571} INFO - 24/01/30 02:52:59 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 33597, None)
[2024-01-30T02:53:01.405+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:01 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-01-30T02:53:01.416+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:01 INFO SharedState: Warehouse path is 'file:/home/juniortemgoua0/DataLake/spark-warehouse'.
[2024-01-30T02:53:03.457+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T02:53:03.457+0000] {spark_submit.py:571} INFO - Date d'exécution  : 2024-01-08
[2024-01-30T02:53:03.458+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T02:53:05.138+0000] {spark_submit.py:571} INFO - Exception in thread "main" org.apache.spark.sql.AnalysisException: [PATH_NOT_FOUND] Path does not exist: hdfs://localhost:9000/data-lake/raw/football_data/competitions/2024-01-08/*.json.
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.errors.QueryCompilationErrors$.dataPathNotExistError(QueryCompilationErrors.scala:1500)
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$3(DataSource.scala:743)
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at org.apache.spark.util.ThreadUtils$.$anonfun$parmap$2(ThreadUtils.scala:380)
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at scala.util.Success.$anonfun$map$1(Try.scala:255)
[2024-01-30T02:53:05.139+0000] {spark_submit.py:571} INFO - at scala.util.Success.map(Try.scala:213)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[2024-01-30T02:53:05.140+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2024-01-30T02:53:05.141+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[2024-01-30T02:53:05.141+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[2024-01-30T02:53:05.141+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[2024-01-30T02:53:05.141+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[2024-01-30T02:53:05.148+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO SparkContext: Invoking stop() from shutdown hook
[2024-01-30T02:53:05.150+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-01-30T02:53:05.171+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO SparkUI: Stopped Spark web UI at http://instance-1.europe-west9-a.c.data-lake-project-409321.internal:4040
[2024-01-30T02:53:05.195+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-01-30T02:53:05.209+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO MemoryStore: MemoryStore cleared
[2024-01-30T02:53:05.209+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO BlockManager: BlockManager stopped
[2024-01-30T02:53:05.218+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-01-30T02:53:05.222+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-01-30T02:53:05.233+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO SparkContext: Successfully stopped SparkContext
[2024-01-30T02:53:05.233+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO ShutdownHookManager: Shutdown hook called
[2024-01-30T02:53:05.234+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO ShutdownHookManager: Deleting directory /tmp/spark-a69acf62-f2be-4860-84e6-dc5c998198d7
[2024-01-30T02:53:05.239+0000] {spark_submit.py:571} INFO - 24/01/30 02:53:05 INFO ShutdownHookManager: Deleting directory /tmp/spark-7c946cb1-c132-4665-b4c8-3e39886dbd21
[2024-01-30T02:53:05.303+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.
[2024-01-30T02:53:05.306+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=manage_data_lake_dag, task_id=local_job_raw_to_formatted, execution_date=20240108T000000, start_date=20240130T025250, end_date=20240130T025305
[2024-01-30T02:53:05.318+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 177 for task local_job_raw_to_formatted (Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.; 18216)
[2024-01-30T02:53:05.354+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-30T02:53:05.369+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-01-30T03:23:11.189+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [queued]>
[2024-01-30T03:23:11.196+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [queued]>
[2024-01-30T03:23:11.196+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 6
[2024-01-30T03:23:11.209+0000] {taskinstance.py:2191} INFO - Executing <Task(SparkSubmitOperator): local_job_raw_to_formatted> on 2024-01-08 00:00:00+00:00
[2024-01-30T03:23:11.215+0000] {standard_task_runner.py:60} INFO - Started process 28753 to run task
[2024-01-30T03:23:11.220+0000] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'manage_data_lake_dag', 'local_job_raw_to_formatted', 'scheduled__2024-01-08T00:00:00+00:00', '--job-id', '176', '--raw', '--subdir', 'DAGS_FOLDER/data_lake_dag.py', '--cfg-path', '/tmp/tmp4w2vopp6']
[2024-01-30T03:23:11.223+0000] {standard_task_runner.py:88} INFO - Job 176: Subtask local_job_raw_to_formatted
[2024-01-30T03:23:11.266+0000] {task_command.py:423} INFO - Running <TaskInstance: manage_data_lake_dag.local_job_raw_to_formatted scheduled__2024-01-08T00:00:00+00:00 [running]> on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T03:23:11.317+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='juniortemgoua0' AIRFLOW_CTX_DAG_ID='manage_data_lake_dag' AIRFLOW_CTX_TASK_ID='local_job_raw_to_formatted' AIRFLOW_CTX_EXECUTION_DATE='2024-01-08T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-01-08T00:00:00+00:00'
[2024-01-30T03:23:11.323+0000] {base.py:83} INFO - Using connection ID 'spark_default' for task execution.
[2024-01-30T03:23:11.325+0000] {spark_submit.py:401} INFO - Spark-Submit cmd: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake
[2024-01-30T03:23:13.904+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:13 INFO SparkContext: Running Spark version 3.5.0
[2024-01-30T03:23:13.910+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:13 INFO SparkContext: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T03:23:13.911+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:13 INFO SparkContext: Java version 17.0.9
[2024-01-30T03:23:14.019+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2024-01-30T03:23:14.152+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceUtils: ==============================================================
[2024-01-30T03:23:14.153+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceUtils: No custom resources configured for spark.driver.
[2024-01-30T03:23:14.154+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceUtils: ==============================================================
[2024-01-30T03:23:14.155+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SparkContext: Submitted application: RawToFormatted
[2024-01-30T03:23:14.188+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2024-01-30T03:23:14.202+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceProfile: Limiting resource is cpu
[2024-01-30T03:23:14.203+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2024-01-30T03:23:14.278+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SecurityManager: Changing view acls to: juniortemgoua0
[2024-01-30T03:23:14.279+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SecurityManager: Changing modify acls to: juniortemgoua0
[2024-01-30T03:23:14.279+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SecurityManager: Changing view acls groups to:
[2024-01-30T03:23:14.280+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SecurityManager: Changing modify acls groups to:
[2024-01-30T03:23:14.281+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users with view permissions: juniortemgoua0; groups with view permissions: EMPTY; users with modify permissions: juniortemgoua0; groups with modify permissions: EMPTY
[2024-01-30T03:23:14.693+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO Utils: Successfully started service 'sparkDriver' on port 38045.
[2024-01-30T03:23:14.735+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SparkEnv: Registering MapOutputTracker
[2024-01-30T03:23:14.800+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SparkEnv: Registering BlockManagerMaster
[2024-01-30T03:23:14.830+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2024-01-30T03:23:14.831+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2024-01-30T03:23:14.835+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2024-01-30T03:23:14.863+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-0c98e461-09ee-4942-afe8-9ccec202fa39
[2024-01-30T03:23:14.888+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2024-01-30T03:23:14.908+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:14 INFO SparkEnv: Registering OutputCommitCoordinator
[2024-01-30T03:23:15.092+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO JettyUtils: Start Jetty 0.0.0.0:4040 for SparkUI
[2024-01-30T03:23:15.180+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2024-01-30T03:23:15.238+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO SparkContext: Added JAR file:/home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar at spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:38045/jars/spark_job_2.12-0.1.0.jar with timestamp 1706584993889
[2024-01-30T03:23:15.337+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Starting executor ID driver on host instance-1.europe-west9-a.c.data-lake-project-409321.internal
[2024-01-30T03:23:15.339+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: OS info Linux, 5.15.0-1049-gcp, amd64
[2024-01-30T03:23:15.340+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Java version 17.0.9
[2024-01-30T03:23:15.350+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Starting executor with user classpath (userClassPathFirst = false): ''
[2024-01-30T03:23:15.351+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Created or updated repl class loader org.apache.spark.util.MutableURLClassLoader@6ed043d3 for default.
[2024-01-30T03:23:15.373+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:38045/jars/spark_job_2.12-0.1.0.jar with timestamp 1706584993889
[2024-01-30T03:23:15.515+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO TransportClientFactory: Successfully created connection to instance-1.europe-west9-a.c.data-lake-project-409321.internal/10.200.0.5:38045 after 78 ms (0 ms spent in bootstraps)
[2024-01-30T03:23:15.532+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Utils: Fetching spark://instance-1.europe-west9-a.c.data-lake-project-409321.internal:38045/jars/spark_job_2.12-0.1.0.jar to /tmp/spark-28870532-0c27-431a-99f0-093ac59cfc72/userFiles-0ce1b48d-3650-4886-b4cf-c9decc9c3b07/fetchFileTemp16241094149701465984.tmp
[2024-01-30T03:23:15.588+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Executor: Adding file:/tmp/spark-28870532-0c27-431a-99f0-093ac59cfc72/userFiles-0ce1b48d-3650-4886-b4cf-c9decc9c3b07/spark_job_2.12-0.1.0.jar to class loader default
[2024-01-30T03:23:15.599+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 37633.
[2024-01-30T03:23:15.599+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO NettyBlockTransferService: Server created on instance-1.europe-west9-a.c.data-lake-project-409321.internal:37633
[2024-01-30T03:23:15.601+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2024-01-30T03:23:15.612+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 37633, None)
[2024-01-30T03:23:15.614+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO BlockManagerMasterEndpoint: Registering block manager instance-1.europe-west9-a.c.data-lake-project-409321.internal:37633 with 434.4 MiB RAM, BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 37633, None)
[2024-01-30T03:23:15.618+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 37633, None)
[2024-01-30T03:23:15.620+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:15 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, instance-1.europe-west9-a.c.data-lake-project-409321.internal, 37633, None)
[2024-01-30T03:23:16.599+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:16 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2024-01-30T03:23:16.610+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:16 INFO SharedState: Warehouse path is 'file:/home/juniortemgoua0/DataLake/spark-warehouse'.
[2024-01-30T03:23:17.641+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T03:23:17.642+0000] {spark_submit.py:571} INFO - Date d'exécution  : 2024-01-08
[2024-01-30T03:23:17.642+0000] {spark_submit.py:571} INFO - ************************************************
[2024-01-30T03:23:18.195+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 WARN FileStreamSink: Assume no metadata directory. Error while looking for metadata directory in the path: hdfs://localhost:9000/data-lake/raw/football_data/competitions/2024-01-08/*.json.
[2024-01-30T03:23:18.196+0000] {spark_submit.py:571} INFO - java.net.ConnectException: Call From instance-1/10.200.0.5 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
[2024-01-30T03:23:18.196+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-01-30T03:23:18.196+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-01-30T03:23:18.196+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-01-30T03:23:18.196+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[2024-01-30T03:23:18.197+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[2024-01-30T03:23:18.197+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-01-30T03:23:18.197+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:828)
[2024-01-30T03:23:18.197+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-01-30T03:23:18.197+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-01-30T03:23:18.198+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-01-30T03:23:18.198+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-01-30T03:23:18.198+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-01-30T03:23:18.198+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy34.getFileInfo(Unknown Source)
[2024-01-30T03:23:18.198+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolTranslatorPB.getFileInfo(ClientNamenodeProtocolTranslatorPB.java:965)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-01-30T03:23:18.199+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy35.getFileInfo(Unknown Source)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.getFileInfo(DFSClient.java:1739)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$29.doCall(DistributedFileSystem.java:1753)
[2024-01-30T03:23:18.200+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$29.doCall(DistributedFileSystem.java:1750)
[2024-01-30T03:23:18.201+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)
[2024-01-30T03:23:18.201+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.getFileStatus(DistributedFileSystem.java:1765)
[2024-01-30T03:23:18.201+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.isDirectory(FileSystem.java:1777)
[2024-01-30T03:23:18.201+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.streaming.FileStreamSink$.hasMetadata(FileStreamSink.scala:54)
[2024-01-30T03:23:18.201+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:366)
[2024-01-30T03:23:18.202+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:229)
[2024-01-30T03:23:18.202+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:211)
[2024-01-30T03:23:18.202+0000] {spark_submit.py:571} INFO - at scala.Option.getOrElse(Option.scala:189)
[2024-01-30T03:23:18.207+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:211)
[2024-01-30T03:23:18.207+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.json(DataFrameReader.scala:362)
[2024-01-30T03:23:18.207+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.DataFrameReader.json(DataFrameReader.scala:341)
[2024-01-30T03:23:18.207+0000] {spark_submit.py:571} INFO - at RawToFormatted$.formattedCompetition(RawToFormatted.scala:39)
[2024-01-30T03:23:18.207+0000] {spark_submit.py:571} INFO - at RawToFormatted$.main(RawToFormatted.scala:26)
[2024-01-30T03:23:18.208+0000] {spark_submit.py:571} INFO - at RawToFormatted.main(RawToFormatted.scala)
[2024-01-30T03:23:18.208+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:23:18.208+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:23:18.208+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:23:18.208+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.JavaMainApplication.start(SparkApplication.scala:52)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.org$apache$spark$deploy$SparkSubmit$$runMain(SparkSubmit.scala:1029)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doRunMain$1(SparkSubmit.scala:194)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.submit(SparkSubmit.scala:217)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.doSubmit(SparkSubmit.scala:91)
[2024-01-30T03:23:18.209+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$$anon$2.doSubmit(SparkSubmit.scala:1120)
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit$.main(SparkSubmit.scala:1129)
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkSubmit.main(SparkSubmit.scala)
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnect(Native Method)
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
[2024-01-30T03:23:18.210+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946)
[2024-01-30T03:23:18.211+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-01-30T03:23:18.211+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-01-30T03:23:18.211+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-01-30T03:23:18.211+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-01-30T03:23:18.211+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-01-30T03:23:18.212+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-01-30T03:23:18.212+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-01-30T03:23:18.212+0000] {spark_submit.py:571} INFO - ... 44 more
[2024-01-30T03:23:18.246+0000] {spark_submit.py:571} INFO - Exception in thread "main" java.net.ConnectException: Call From instance-1/10.200.0.5 to localhost:9000 failed on connection exception: java.net.ConnectException: Connection refused; For more details see:  http://wiki.apache.org/hadoop/ConnectionRefused
[2024-01-30T03:23:18.246+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2024-01-30T03:23:18.246+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
[2024-01-30T03:23:18.246+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2024-01-30T03:23:18.246+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
[2024-01-30T03:23:18.247+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
[2024-01-30T03:23:18.247+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapWithMessage(NetUtils.java:913)
[2024-01-30T03:23:18.247+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.wrapException(NetUtils.java:828)
[2024-01-30T03:23:18.247+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1616)
[2024-01-30T03:23:18.247+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1558)
[2024-01-30T03:23:18.248+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1455)
[2024-01-30T03:23:18.248+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)
[2024-01-30T03:23:18.248+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)
[2024-01-30T03:23:18.249+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy34.getListing(Unknown Source)
[2024-01-30T03:23:18.249+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolTranslatorPB.getListing(ClientNamenodeProtocolTranslatorPB.java:688)
[2024-01-30T03:23:18.249+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2024-01-30T03:23:18.249+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[2024-01-30T03:23:18.249+0000] {spark_submit.py:571} INFO - at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2024-01-30T03:23:18.250+0000] {spark_submit.py:571} INFO - at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[2024-01-30T03:23:18.250+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)
[2024-01-30T03:23:18.250+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)
[2024-01-30T03:23:18.250+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)
[2024-01-30T03:23:18.250+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)
[2024-01-30T03:23:18.251+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)
[2024-01-30T03:23:18.251+0000] {spark_submit.py:571} INFO - at jdk.proxy2/jdk.proxy2.$Proxy35.getListing(Unknown Source)
[2024-01-30T03:23:18.251+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.listPaths(DFSClient.java:1702)
[2024-01-30T03:23:18.251+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DFSClient.listPaths(DFSClient.java:1686)
[2024-01-30T03:23:18.251+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.listStatusInternal(DistributedFileSystem.java:1100)
[2024-01-30T03:23:18.252+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.access$600(DistributedFileSystem.java:147)
[2024-01-30T03:23:18.252+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$24.doCall(DistributedFileSystem.java:1175)
[2024-01-30T03:23:18.252+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem$24.doCall(DistributedFileSystem.java:1172)
[2024-01-30T03:23:18.252+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)
[2024-01-30T03:23:18.253+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.hdfs.DistributedFileSystem.listStatus(DistributedFileSystem.java:1182)
[2024-01-30T03:23:18.254+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.listStatus(Globber.java:128)
[2024-01-30T03:23:18.254+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.doGlob(Globber.java:291)
[2024-01-30T03:23:18.254+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.Globber.glob(Globber.java:202)
[2024-01-30T03:23:18.255+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.fs.FileSystem.globStatus(FileSystem.java:2124)
[2024-01-30T03:23:18.255+0000] {spark_submit.py:571} INFO - at org.apache.spark.deploy.SparkHadoopUtil.globPath(SparkHadoopUtil.scala:238)
[2024-01-30T03:23:18.255+0000] {spark_submit.py:571} INFO - at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$3(DataSource.scala:737)
[2024-01-30T03:23:18.255+0000] {spark_submit.py:571} INFO - at org.apache.spark.util.ThreadUtils$.$anonfun$parmap$2(ThreadUtils.scala:380)
[2024-01-30T03:23:18.255+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.util.Success.$anonfun$map$1(Try.scala:255)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.util.Success.map(Try.scala:213)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[2024-01-30T03:23:18.256+0000] {spark_submit.py:571} INFO - at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1395)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
[2024-01-30T03:23:18.257+0000] {spark_submit.py:571} INFO - at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - Caused by: java.net.ConnectException: Connection refused
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnect(Native Method)
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946)
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:205)
[2024-01-30T03:23:18.258+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:586)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:711)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:833)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client$Connection.access$3800(Client.java:414)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.getConnection(Client.java:1677)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - at org.apache.hadoop.ipc.Client.call(Client.java:1502)
[2024-01-30T03:23:18.259+0000] {spark_submit.py:571} INFO - ... 43 more
[2024-01-30T03:23:18.267+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO SparkContext: Invoking stop() from shutdown hook
[2024-01-30T03:23:18.267+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO SparkContext: SparkContext is stopping with exitCode 0.
[2024-01-30T03:23:18.279+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO SparkUI: Stopped Spark web UI at http://instance-1.europe-west9-a.c.data-lake-project-409321.internal:4040
[2024-01-30T03:23:18.298+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2024-01-30T03:23:18.314+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO MemoryStore: MemoryStore cleared
[2024-01-30T03:23:18.315+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO BlockManager: BlockManager stopped
[2024-01-30T03:23:18.323+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO BlockManagerMaster: BlockManagerMaster stopped
[2024-01-30T03:23:18.327+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2024-01-30T03:23:18.337+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO SparkContext: Successfully stopped SparkContext
[2024-01-30T03:23:18.338+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO ShutdownHookManager: Shutdown hook called
[2024-01-30T03:23:18.339+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-1fbe116a-fd92-4bbd-8a2f-a10ac01515ac
[2024-01-30T03:23:18.342+0000] {spark_submit.py:571} INFO - 24/01/30 03:23:18 INFO ShutdownHookManager: Deleting directory /tmp/spark-28870532-0c27-431a-99f0-093ac59cfc72
[2024-01-30T03:23:18.402+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/operators/spark_submit.py", line 174, in execute
    self._hook.submit(self._application)
  File "/home/juniortemgoua0/DataLake/venv/lib/python3.8/site-packages/airflow/providers/apache/spark/hooks/spark_submit.py", line 502, in submit
    raise AirflowException(
airflow.exceptions.AirflowException: Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.
[2024-01-30T03:23:18.405+0000] {taskinstance.py:1138} INFO - Marking task as UP_FOR_RETRY. dag_id=manage_data_lake_dag, task_id=local_job_raw_to_formatted, execution_date=20240108T000000, start_date=20240130T032311, end_date=20240130T032318
[2024-01-30T03:23:18.417+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 176 for task local_job_raw_to_formatted (Cannot execute: spark-submit --master local[*] --conf spark.airflow.execution_date=2024-01-08 --name arrow-spark --class RawToFormatted --queue root.default --deploy-mode client /home/juniortemgoua0/DataLake/jobs/processes/scala/spark_process/target/scala-2.12/spark_job_2.12-0.1.0.jar hdfs://localhost:9000/data-lake. Error code is: 1.; 28753)
[2024-01-30T03:23:18.438+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-30T03:23:18.454+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
